packages/sdk/src/Campaigns.ts

```diff
@@ -18,43 +20,54 @@ export class Campaigns {
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#create-campaign
 	 */
-	public create(campaign: $FixMe) {
-		return this.client.post('/campaigns', campaign)
+	public create(campaign: T.CampaignsCreateCampaign) {
+		return this.client.post<T.CampaignsCreateCampaignResponse>('/campaigns', campaign)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#update-campaign
 	 */
-	public update(nameOrId: $FixMe, campaign: $FixMe) {
-		return this.client.put(`/campaigns/${encode(nameOrId)}`, campaign)
+	public update(nameOrId: string, campaign: T.CampaignsUpdateCampaign) {
+		return this.client.put<T.CampaignsUpdateCampaignResponse>(`/campaigns/${encode(nameOrId)}`, campaign)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#get-campaign
 	 */
-	public get(name: $FixMe) {
-		return this.client.get(`/campaigns/${encode(name)}`)
+	public get(name: string) {
+		return this.client.get<T.CampaignsGetCampaignResponse>(`/campaigns/${encode(name)}`)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#delete-campaign
 	 */
-	public delete(name: $FixMe, params?: { force: boolean }) {
-		return this.client.delete(`/campaigns/${encode(name)}`, { force: !!params?.force })
+	public delete(name: string, params?: T.CampaignsDeleteParams) {
+		return this.client.delete(`/campaigns/${encode(name)}`, params)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#add-voucher-to-campaign
 	 */
-	public addVoucher(campaignName: string, body: $FixMe = {}) {
-		return this.client.post(`/campaigns/${encode(campaignName)}/vouchers`, body)
+
+	public addVoucher(name: string, params: T.CampaignsAddVoucherParams = {}) {
+		return this.client.post<T.CampaignsAddVoucherResponse>(`/campaigns/${encode(name)}/vouchers`, params)
+	}
+	/**
+	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#add-voucher-with-certain-code-to-campaign
+	 */
+
+	public addCertainVoucher(name: string, code: string, params: T.CampaignsAddCertainVoucherParams = {}) {
+		return this.client.post<T.CampaignsAddCertainVoucherResponse>(
+			`/campaigns/${encode(name)}/vouchers/${encode(code)}`,
+			params,
+		)
 	}
 	/**
 	 * @see https://docs.voucherify.io/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#import-vouchers
 	 */
-	public importVouchers(campaignName: string, vouchers: $FixMe) {
+	public importVouchers(campaignName: string, vouchers: T.CampaignsImportVouchers[]) {
 		return this.client.post(`/campaigns/${encode(campaignName)}/import`, vouchers)
 	}
 	/**
 	 * @see https://docs.voucherify.io/v2017-04-20/reference?utm_source=github&utm_medium=sdk&utm_campaign=acq#list-campaigns
 	 */
-	public list(params: $FixMe) {
+	public list(params: T.CampaignsListParams) {
```

Comment:
Please make it optional via `={}`, see old SDK:
https://github.com/voucherifyio/voucherify-nodejs-sdk#list-campaigns